g2core/report.cpp

```diff
@@ -337,17 +337,18 @@ stat_t sr_status_report_callback()         // called by controller dispatcher
         return (STAT_NOOP);
     }
 
-   // don't send an SR if you the planner is experiencing a time constraint
-   if (!mp_is_phat_city_time()) {
+    // don't send an SR if you the planner is experiencing a time constraint
+    if (!mp_is_phat_city_time()) {
         if (++sr.throttle_counter != SR_THROTTLE_COUNT) {
             return (STAT_NOOP);
         }
         sr.throttle_counter = 0;
     }
 
-    sr.status_report_request = SR_OFF;
+    srVerbosity req_sr_verbosity = sr.status_report_request;
```

Comment:
In case `sr.status_report_request` changes type, we can use `auto` here as we don’t really care about the type, and helps with refactors.

```c++
auto req_sr_verbosity = sr.status_report_request;
```